config/defaults.cmake

```diff
@@ -19,6 +19,7 @@ if (NOT CMAKE_BUILD_TYPE)
 endif()
 
 # MFEM options. Set to mimic the default "defaults.mk" file.
+option(BUILD_SHARED_LIBS "Enable shared library build of MFEM" OFF)
```

Comment:
This option is built into CMake, but no harm in making it explicit
See: https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html